library/src/main/api14/com/google/android/cameraview/Camera1.java

```diff
@@ -227,14 +230,18 @@ public void onAutoFocus(boolean success, Camera camera) {
     }
 
     void takePictureInternal() {
-        mCamera.takePicture(null, null, null, new Camera.PictureCallback() {
-            @Override
-            public void onPictureTaken(byte[] data, Camera camera) {
-                mCallback.onPictureTaken(data);
-                camera.cancelAutoFocus();
-                camera.startPreview();
-            }
-        });
+        try {
+            mCamera.takePicture(null, null, null, new Camera.PictureCallback() {
```

Comment:
Thanks for your suggestion. Could you please specify what kind of effect will this change have?